tutorials/README.md

```diff
@@ -0,0 +1,29 @@
+# Tutorials
+
+This area is comprised of tutorials that focus on various aspects of the AoU workbench code base, such as: code organization, build processes, tools & technologies, back end API, database and front end architecure.
```

Comment:
There is an existing directory that seemingly has the same purpose as this one. I'd suggest merging them: https://github.com/all-of-us/workbench/tree/master/api/docs/tutorials . We can potentially lift that `docs` folder up to the top-level, if that better matches the intent here, or else just have parallel directories under api/ui.